questionnaire/questionnaire-service.js

```diff
@@ -147,7 +147,10 @@ function createQuestionnaireService({
                 const notifyService = createNotifyService({logger});
 
                 if (task.type === 'sendEmail') {
-                    if (pointer.has(questionnaire, task.templatePlaceholderMap.emailAddress)) {
+                    if (
```

Comment:
Can we cache the value as per the replace-json-pointer "has" check?

Reply:
np